### PR TITLE
Don't append ansible-service-broker suffix without a check

### PR DESCRIPTION
**Pull request by dymurray**

### CODE REVIEW

File: src/apb/engine.py [hunk 1]
```diff
@@ -726,8 +726,6 @@ def retrieve_test_result(name, namespace):
 def broker_request(broker, service_route, method, **kwargs):
     if broker is None:
         broker = get_asb_route()
-    else:
-        broker = "%s/ansible-service-broker" % broker
```
Comment (dymurray) [inline src/apb/engine.py:730]: From apb push we were already setting broker so this was appending a second suffix.

File: src/apb/engine.py [hunk 2]
```diff
@@ -1047,8 +1045,6 @@ def cmdrun_push(**kwargs):
     blob = base64.b64encode(spec)
     data_spec = {'apbSpec': blob}
     broker = kwargs["broker"]
-    if broker is None:
-        broker = get_asb_route()
```
Comment (paolobueno) [inline src/apb/engine.py]: @dymurray is this removal correct here? maybe this is what's making `--broker` required

Comment (paolobueno) [reply src/apb/engine.py]: Ah no, it seems it's being called from `broker_request()`

Comment (dymurray) [reply src/apb/engine.py]: @paolobueno Yeah there seems to be something that the openshift restclient is complaining about when running `get_asb_route()` from `broker_request()`.

**Review by rthallisey — APPROVED** [review]

**Pull request merged**